redash_toolbelt/client.py

```diff
@@ -73,17 +73,21 @@ def dashboard(self, slug):
         return self._get("api/dashboards/{}".format(slug)).json()
 
     def create_query(self, query_json):
+        """POST api/queries with query_json object"""
```

Comment:
```suggestion
        """POST api/queries with query_json object
        
        This method calls redash.handlers.queries.QueryListResource.post
        
        Args:
          query_json: An object representing a new query to be created. Must contain the following keys:
            - query: the query text
            - data_source_id: a valid data source id
            - title: the name of this query"""
```

An example of a more detailed docstring, per my comment.